continuous-benchmarking/www/server.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const readline = require('readline');
+const fs = require('fs');
+const express = require('express');
```

Comment:
:cherries: I needed to run `npm install -g express`. Maybe nice to have this info in readme (or include in the setup script )